sample-data.n3

```diff
@@ -250,7 +257,26 @@
 <http://vivo.mydomain.edu/individual/n8066>
         a             vivo:Authorship ;
         vivo:relates  <http://vivo.mydomain.edu/individual/n3728> , <http://vivo.mydomain.edu/individual/n128> .
-
+        
+<http://vivo.mydomain.edu/individual/n7124>
+        a             vivo:Authorship ;
+        vivo:rank     "1"^^xsd:int ;
+        vivo:relates  <http://vivo.mydomain.edu/individual/n7578> , <http://vivo.mydomain.edu/individual/n6870> .
+        
+<http://vivo.mydomain.edu/individual/n2809>
+        a             vivo:Authorship ;
+        vivo:rank     "2"^^xsd:int ;
+        vivo:relates  <http://vivo.mydomain.edu/individual/n7578> , <http://vivo.mydomain.edu/individual/n1111> .
+        
+<http://vivo.mydomain.edu/individual/n1111>
+		a             vcard:Individual ;
```

Comment:
Can you please align this and the next one individual layout with the rest of the document? Indentation is not the same.